web/ts/watch.ts

```diff
@@ -19,8 +19,9 @@ let retryInt = 5000; //retry connecting to websocket after this timeout
 const pageloaded = new Date();
 
 function startWebsocket() {
+    const wsProto = window.location.protocol.indexOf("https") !== -1 ? `wss://` : `ws://`;
```

Comment:
Just an idea but you could make it more simple: `window.location.protocol == "https:"`.
https://developer.mozilla.org/en-US/docs/Web/API/Location/protocol

Reply:
You're right, that makes much more sense, I changed it :)